.github/workflows/scala_container_publishing.yml

```diff
@@ -54,24 +58,18 @@ jobs:
       - name: Log in to Github Container Registry
         uses: docker/login-action@v3
         with:
-          registry: ${{ env.REGISTRY }}
+          registry: ${{ env.GHCR_REGISTRY }}
           username: ${{ github.actor }}
           password: ${{ secrets.GH_TOKEN }}
       - name: Extract metadata (tags, labels) for Docker
         id: meta
         uses: docker/metadata-action@v5
         with:
           images: |
-            ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+            ${{ env.GHCR_REGISTRY }}/${{ env.IMAGE_NAME }}
+            ${{ env.DOCKERHUB_NAMESPACE }}/${{ github.event.repository.name }}
```

Comment:
Does env.IMAGE_NAME expand differently than github.event.repository.name? I'm not entirely sure what these work out to just reading this.